### ISSUE TITLE
Missing $RESTPWFILE and $ADMINPWFILE

### ISSUE THREAD

**Issue by yonasadiel** [DOMjudge/domjudge#670]
### Description of the problem
On these codes,
https://github.com/DOMjudge/domjudge/blob/144aba77fd3092f23a1e6a78e612d9d2db686cb7/sql/dj_setup_database.in#L220-L223
However, in commit 144aba77fd3092f23a1e6a78e612d9d2db686cb7, the variable is removed.
```
RESTPWFILE="$ETCDIR/restapi.secret"	
ADMINPWFILE="$ETCDIR/initial_admin_password.secret"
```

### Your environment
- DOMjudge release 7.1.1
- Ubuntu 18.04
- Apache

### Steps to reproduce
(basic installation)
- `make dist`
- `./configure --prefix=$HOME/domjudge`
- `make domserver && make install-domserver`
- `cd $HOME/domjudge/domserver`
- `rm etc/dbpasswords.secret`
- `bin/dj_setup_database genpass`

### Expected behaviour
On the last command, the password should be generated for db, restapi, and admin.

### Actual behaviour
It fails on creating restapi password, because the variable is undefined

### Any other information that you want to share?
-

**Comment by eldering** [comment]
Should be fixed in the master and 7.1 branch. However, I noticed that the way we generate and use passwords is quite fragile. For example, if you regenerate the DB passwords, then the copy in `webapp/.env.local*` is not automatically updated too. Sideways related #650.

**Comment by eldering** [comment]
Closed this ticket, further problems can be tracked in #673.
